src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitTransaction> fruitTransactions = new ArrayList<>();
```

Comment:
On the Storage fruits are kept, not Transactions
```suggestion
    public static final Map<String, Integer> fruits = new HashMap<>();
```